### ISSUE TITLE
Proxy not saved to zos network file during  `zos create` if you fail init method

### ISSUE THREAD

**Issue by ylv-io** [zeppelinos/zos#993]
![image](https://user-images.githubusercontent.com/3285000/59715156-ea0cf200-921b-11e9-82ca-e9684e9e0c72.png)

**Comment by spalladino** [comment]
I think this is actually not a bug. The proxy is never deployed if initialization fails, since deployment and initialization are atomic (they all happen on the same tx). That said, we could display a nicer error.

**Comment by ylv-io** [comment]
It says `Deployed  ProxyAdmin at ....` that is clear misinformation then.

**Comment by spalladino** [comment]
That's the `ProxyAdmin` (not the Proxy) which gets deployed the first time you create a proxy on that network, and that one does get deployed (as far as I understood). It **is** confusing though that one is a ProxyAdmin and the other a Proxy.

**Comment by ylv-io** [comment]
You are right! The fact that I got tricked leads me to belive that average dev starting to use platform will have no idea about ProxyAdmin/Proxy difference.

**Comment by ylv-io** [comment]
I think we should then explicitly say that contract has not been deployed. That would be important for mainnet deployments.

**Comment by spalladino** [comment]
If initialization fails due to a parser error (as in the screenshot) or any other reason, notify the user and be explicit that the instance was **not** created.
